src/js/common/polyfills.js

```diff
@@ -24,3 +24,13 @@ if (!Modernizr.fetch) {
 
 // This polyfill has its own test logic so no need to conditionally require.
 require('polyfill-function-prototype-bind');
+
+// Disable scrolling on input[type="number"] because it was changing the value unexpectedly
+// NOTE: This also just stops the user from scrolling when hovering over a number
+//  input that has focus. This isn't ideal, but until a solution is found to that,
+//  this is what we've got.
+document.addEventListener('mousewheel', (event) => {
```

Comment:
I think always adding it on `document` is fine until we have a reason not to. But maybe we should just put this in common/index.js? It's not really a polyfill. I don't have a strong opinion on that.